lib/park.rb

```diff
@@ -0,0 +1,30 @@
+class Park
+  attr_reader :name, :entrance_fee, :vehicles, :revenue
```

Comment:
```suggestion
  attr_reader :name, :admission_price, :vehicles, :revenue
```

The interaction pattern asks us to call this method `admission_price`, not `entrance_fee`. It seems like a small thing to nitpick, but once you start working on larger projects and teams, it can be pretty important to make sure you're following the "contract" or defined requirements. Not doing so can cause bugs further down the line if folks on other teams are expecting this to be called `admission_price` instead.